inlong-sort/sort-connectors/kafka/src/main/java/org/apache/inlong/sort/kafka/table/KafkaInLongOptions.java

```diff
@@ -22,8 +22,8 @@
 import org.apache.flink.configuration.ConfigOptions;
 
 /** Option utils for Kafka table source sink. */
-public class KafkaOptions {
-    private KafkaOptions() {
+public class KafkaInLongOptions {
```

Comment:
This refactoring doesn't make sense, it is recommended to keep it as it is.

Reply:
OK, 3q